packages/in_app_purchase/in_app_purchase_storekit/lib/src/store_kit_wrappers/sk_product_wrapper.dart

```diff
@@ -167,6 +167,23 @@ enum SKProductDiscountPaymentMode {
   unspecified,
 }
 
+/// Dart wrapper around StoreKit's [SKProductDiscountType]
+/// (https://developer.apple.com/documentation/storekit/skproductdiscounttype?language=objc)
+///
+/// This is used as a property in the [SKProductDiscountWrapper].
+/// The values of the enum options are matching the [SKProductDiscountType]'s
+/// values. Should there be an update or addition in the
+/// [SKProductDiscountType], this need to be updated to match.
```

Comment:
"Should there be an update or addition in the [SKProductDiscountType], this need to be updated to match." Sounds more like for implementers, not users.
Maybe we can move this sentence to a comment instead of dart doc:
```
// Should there be an update or addition in the
// [SKProductDiscountType], this need to be updated to match.
```